app/scripts/views/sign_up.js

```diff
@@ -183,6 +183,8 @@ define(function (require, exports, module) {
         signinUri: this.broker.transformLink('/signin')
       };
 
+      this.setShowSyncSuggestion(context);
```

Comment:
@larissagaulia ah sorry didn't test it enough, try doing what the `back-mixin.js` is doing like this:

```diff
diff --git a/app/scripts/templates/sign_up.mustache b/app/scripts/templates/sign_up.mustache
index 80cd456..07dd914 100644
--- a/app/scripts/templates/sign_up.mustache
+++ b/app/scripts/templates/sign_up.mustache
@@ -15,7 +15,8 @@
     {{#error}}
       <div class="error visible">{{ error }}</div>
     {{/error}}
-
+    {{sup}}
+    {{mixin}}
     {{^error}}
     <div class="error"></div>
     <div class="success"></div>
diff --git a/app/scripts/views/mixins/signin-mixin.js b/app/scripts/views/mixins/signin-mixin.js
index 895ee11..4072bcf 100644
--- a/app/scripts/views/mixins/signin-mixin.js
+++ b/app/scripts/views/mixins/signin-mixin.js
@@ -8,6 +8,7 @@ define(function (require, exports, module) {
   'use strict';
 
   var AuthErrors = require('lib/auth-errors');
+  var BaseView = require('views/base');
   var p = require('lib/promise');
   var VerificationMethods = require('lib/verification-methods');
   var VerificationReasons = require('lib/verification-reasons');
@@ -58,6 +59,19 @@ define(function (require, exports, module) {
         });
     },
 
+    /**
+     * Monkey patch BaseView.prototype.getContext to return a context
+     * @method getContext
+     * @returns {Object}
+     */
+    getContext: function () {
+      var context = BaseView.prototype.getContext.call(this);
+
+      context.mixin = 'mixin';
+
+      return context;
+    },
+
     onSignInSuccess: function (account) {
       if (! account.get('verified')) {
         var verificationMethod = account.get('verificationMethod');
diff --git a/app/scripts/views/sign_up.js b/app/scripts/views/sign_up.js
index 0533355..0e1a4ff 100644
--- a/app/scripts/views/sign_up.js
+++ b/app/scripts/views/sign_up.js
@@ -180,7 +180,8 @@ define(function (require, exports, module) {
         serviceName: relier.get('serviceName'),
         shouldFocusEmail: autofocusEl === 'email',
         shouldFocusPassword: autofocusEl === 'password',
-        signinUri: this.broker.transformLink('/signin')
+        signinUri: this.broker.transformLink('/signin'),
+        sup: 'hi'
       };
 
       return context;
```